.circleci/verbatim-sources/job-specs/pytorch-job-specs.yml

```diff
@@ -15,13 +15,6 @@ jobs:
         no_output_timeout: "1h"
         command: |
           set -e
-          # TODO: Remove this after we figure out why rocm tests are failing
-          if [[ "${DOCKER_IMAGE}" == *rocm3.5* ]]; then
-            export DOCKER_TAG="ab1632df-fa59-40e6-8c23-98e004f61148"
-          fi
-          if [[ "${DOCKER_IMAGE}" == *rocm3.7* ]]; then
-            export DOCKER_TAG="1045c7b891104cb4fd23399eab413b6213e48aeb"
-          fi
```

Comment:
Feel free to delete 3.5, but specify a valid rocm3.8 DOCKER_TAG

Reply:
Used same docker tag as in current ossci-job-dsl repo for rocm 3.9 image.